### ISSUE TITLE
2.2.0 Release : New icons are not correctly displayed

### ISSUE THREAD

**Issue by kevzlou7979** [google/material-design-icons#311]
Hi guys,

I have a problem rendering the icons on 2.2.0 to display it using a self hosted css / iconfont resources:
I have downloaded the zip files https://github.com/google/material-design-icons/archive/2.2.0.zip and set it up on my local.

<b>This code : </b>
```html

<i class="material-icons">euro_symbol</i>
	<i class="material-icons">polymer</i>

```

<b> Will generate </b>
![image](https://cloud.githubusercontent.com/assets/3138071/13099737/3bd500b2-d570-11e5-95dd-e2322c18b12c.png)

<b> Here is the inspect dev tool </b>
![image](https://cloud.githubusercontent.com/assets/3138071/13099742/4f07a1d0-d570-11e5-9fc7-35215d28a46b.png)

**Comment by Vayvala** [comment]
+1

**Comment by TroodoNmike** [comment]
Same problem. I went to tagged version of 2.2 and then into iconfont folder. MaterialIcons-Regular.woff2 file seems be missing 2.2 new icons (in chrome does not appear). File .woff seems to work which looks ok in Safari.

**Comment by scottie-33** [comment]
Same problem. Internet Explorer looks also ok.

**Comment by chris1302** [comment]
+1

**Comment by ccloli** [comment]
+1, _first_page_ and _last_page_ are also not displayed when **woff2** font was included in css (woff, ttf and eot work fine).

**Comment by adrchw** [comment]
+1, Probably all new icons are not displayed when woff2 was included.

**Comment by shyndman** [comment]
Fixed in #337
